### PR TITLE
DBZ 2132 | DBZ 2113 | DBZ 2114 OpenShift Tests for SQL Server & Mongo DB

### PR DESCRIPTION
**Pull request by jcechace**
Contains required refactorings and improvement for MySQL and Postgres, as well as added tests for SQL Server and Mongo.

### CODE REVIEW

File: debezium-testing/debezium-testing-openshift/src/main/java/io/debezium/testing/openshift/tools/databases/DatabaseController.java [hunk 1]
```diff
@@ -55,30 +52,31 @@ public String getDatabaseUrl() {
...
     }
 
-    public void executeStatement(String database, String username, String password, String sql) throws SQLException {
-        try (Connection con = DriverManager.getConnection(getDatabaseUrl(), username, password)) {
-            con.setCatalog(database);
-            Statement stmt = con.createStatement();
-            stmt.execute(sql);
-        }
-    }
+    // public void executeStatement(String database, String username, String password, String command) throws SQLException {
```
Comment (jpechane) [inline debezium-testing/debezium-testing-openshift/src/main/java/io/debezium/testing/openshift/tools/databases/DatabaseController.java:58]: The dead code should be removed?

Comment (jcechace) [reply debezium-testing/debezium-testing-openshift/src/main/java/io/debezium/testing/openshift/tools/databases/DatabaseController.java:58]: Certainly... removing

**Comment by jcechace** [comment]
>  Would it make sense , as a follow-up, to refactor it in a way that there will be a base class containing the test and every connector test would just derive from it?

You are correct. Tests are pretty much the same after the refactoring. I'm thinking about moving them into some common super-class. However, I'd like to wait a bit so we know there are no gotchas in these tests which would subsequently lead to moving this back to subclasses.

**Comment by gunnarmorling** [comment]
woohoo, thanks, @jcechace!

**Pull request merged**